app/javascript/retrospring/initializers/share.ts

```diff
@@ -0,0 +1,20 @@
+import $ from 'jquery';
+
+export default (): void => {
+    document.addEventListener('turbolinks:load', function () {
+        if (navigator.share) {
+            document.body.classList.add('cap-web-share')
+            $(document).on('click', 'button[name=ab-share]', function () {
+                const card = $(this).closest('.card')
+
+                navigator.share({
+                    url: card.find('.answerbox__answer-date a')[0].href
+                }).then(() => {
+                    // do nothing, prevents exception from being thrown
+                }).catch(() => {
+                    // do nothing, prevents exception from being thrown
+                })
+            })
+        }
+    })
+}
```

Comment:
* Initializers are meant to set up larger global features (Bootstrap, jQuery, etc. pp.), this is a single feature.
* The code doesn't need to be initialized if no `button[name=ab-share]` is present (save code execution)
* **Don't use jQuery!** It's planned to be removed (and it already is in a branch)

You can use the [`feature/webpacker` branch](https://github.com/Retrospring/retrospring/tree/feature/webpacker/app/javascript) as a reference on how I implemented features before!